app/ml_models/rnn/rnn_model.py

```diff
@@ -24,3 +24,26 @@ def forward(self, input, hidden):
 
     def initHidden(self):
         return torch.zeros(1, self.hidden_size)
+
+
+class RNNAnna(nn.Module):
+    def __init__(
+        self,
+        vocab_size,
+        hidden_size,
+        train_embeddings=False,
+    ):
+        super(RNNAnna, self).__init__()
+        self._embedding = nn.Embedding(vocab_size, vocab_size)
+        self._embedding.weight.data = torch.eye(vocab_size)
+        self._embedding.weight.requires_grad = train_embeddings
+
+        self.lstm = nn.RNN(vocab_size, hidden_size, 1, batch_first=False)
```

Comment:
Would lstm be appropriate naming here given that we're technically using its parent

Reply:
Yes